index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="https://kit.fontawesome.com/8d28ec52c7.js" crossorigin="anonymous"></script>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="/style.css">
+    </head>
+    <body>
+       <div class="container">
+           <div class="container-top">
+               <div class="owner">John Doe</div>
+               <div class="menu-button"><i class="fa fa-bars"></i></div>
+            </div>
+            <div class="name">Hey There. I'm Leonard</div>
+            <div class="my-title">I am a Software Developer</div>
+            <div class="info">I can help you build a product,feature or website.Look through some of my work and experience! if you like what you see and have a project you need coded dont hesitate to contact me</div>
+            <div class="socials">
+                <i class="fa fa-github fa-2x" aria-hidden="true"></i>
+                <i class="fa fa-twitter fa-2x" aria-hidden="true"></i>
+                <i class="fa fa-linkedin fa-2x" aria-hidden="true"></i>
+                <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
+            </div>
```

Comment:
- [ ] The icons are each expected to be wrapped inside the `<a>` tags and also wrapped inside the `<ul>`  and `<li>` tags to enhance semantics.

Reply:
resolved 👍